### PR TITLE
Survey template list

### PR DESCRIPTION
**Pull request by SteRiccio**
## Description

- #### Added Surveys common component (used by SurveyList and SurveyTemplateList view components)

- #### Added SurveyTemplateList component and added to Home views
  The only difference between the list of surveys is in the title in the top left corner and the lack of the Status column.
   
## Types of changes

- [ ] Docs change
- [ ] Refactoring
- [ ] Dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How has this been tested

Changes have been tested only locally.
Templates creation will be added later on and E2E tests will be added together with that feature.

#### Do you consider this PR needs further testing?

- [x] No
- [ ] Yes

## UI changes

Screenshot_2021-04-13_14-54-55

Screenshot_2021-04-13_10-35-02

### CODE REVIEW

**Comment by minotogna** [comment]
Task linked: [CU-g10xaf Survey template list](https://app.clickup.com/t/g10xaf)

**Comment by minotogna** [comment]
can u update a screenshot with User menu as well please?

File: webapp/components/Surveys/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export { Surveys } from './Surveys'
```
Comment (minotogna) [inline webapp/components/Surveys/index.js:1]: I would move this under webapp/components/survey/Surveys/

Comment (SteRiccio) [reply webapp/components/Surveys/index.js:1]: ok

**Review by minotogna — APPROVED** [review]

**Pull request merged**